### PR TITLE
[Refactor] modify and format dinky-pg.sql

### PR DESCRIPTION
**Pull request by ningyanhui**
## Purpose of the pull request
modify dinky-pg.sql

## Brief change log
1. add dinky-pg.sql default value,comment,trigger for update update_time
2. format dinky-pg.sql

## Verify this pull request

This pull request is code cleanup without any test coverage.

This change added tests and can be verified as follows:

### CODE REVIEW

File: script/sql/dinky-pg.sql [hunk 1]
```diff
@@ -2204,1748 +2303,1787 @@ WITH
...
+COMMENT
+ON COLUMN dinky_job_history.checkpoints_json IS '检查点 json';
+COMMENT
+ON COLUMN dinky_job_history.checkpoints_config_json IS '检查点配置 json';
+COMMENT
+ON COLUMN dinky_job_history.config_json IS '配置';
+COMMENT
+ON COLUMN dinky_job_history.cluster_json IS '集群实例配置';
+COMMENT
+ON COLUMN dinky_job_history.cluster_configuration_json IS '集群配置';
```
Comment (aiwenmo) [inline script/sql/dinky-pg.sql]: Please use English.

**Comment by Zzm0809** [comment]
must be have primary key

File: script/sql/dinky-pg.sql [hunk 2]
```diff
@@ -2208,1669 +2309,2030 @@ VALUES (248, 'Variable', 'SQL_TEMPLATE', 'FlinkSql', 'add-customjar',
...
+    BEFORE UPDATE
+    ON dinky_sys_menu
+    FOR EACH ROW
+EXECUTE FUNCTION trigger_set_timestamp();
+
+-- ----------------------------
+-- Records of dinky_sys_menu
+-- ----------------------------
+BEGIN;
+insert into dinky_sys_menu
```
Comment (Zzm0809) [inline script/sql/dinky-pg.sql]: 这条记录先注释掉

File: script/sql/dinky-pg.sql [hunk 3]
```diff
@@ -2208,1669 +2309,2030 @@ VALUES (248, 'Variable', 'SQL_TEMPLATE', 'FlinkSql', 'add-customjar',
...
+        'AlertOutlined', 'C', 0, 48, '2023-08-11 14:06:55', '2023-09-26 15:02:23', null);
+insert into dinky_sys_menu
+values (30, 9, '集群配置', '/registration/cluster/config', './RegCenter/Cluster/Configuration',
+        'registration:cluster:config', 'SettingOutlined', 'C', 0, 31, '2023-08-11 14:06:55', '2023-09-26 14:57:57',
+        null);
+insert into dinky_sys_menu
+values (31, 12, '告警实例', '/registration/alert/instance', './RegCenter/Alert/AlertInstance',
+        'registration:alert:instance', 'AlertFilled', 'C', 0, 44, '2023-08-11 14:06:55', '2023-09-26 15:01:42', null);
+insert into dinky_sys_menu
+values (32, 1, '作业监控', '/home/jobOverView', 'JobOverView', 'home:jobOverView', 'AntCloudOutlined', 'F', 0, 2,
```
Comment (Zzm0809) [inline script/sql/dinky-pg.sql]: 这条记录先注释掉

File: script/sql/dinky-pg.sql [hunk 4]
```diff
@@ -2208,1669 +2309,2030 @@ VALUES (248, 'Variable', 'SQL_TEMPLATE', 'FlinkSql', 'add-customjar',
...
+        '2023-08-15 16:52:59', '2023-09-26 14:48:50', null);
+insert into dinky_sys_menu
+values (33, 1, '数据开发', '/home/devOverView', 'DevOverView', 'home:devOverView', 'AimOutlined', 'F', 0, 3,
```
Comment (Zzm0809) [inline script/sql/dinky-pg.sql]: 这条记录先注释掉

File: script/sql/dinky-pg.sql [hunk 5]
```diff
@@ -2208,1669 +2309,2030 @@ VALUES (248, 'Variable', 'SQL_TEMPLATE', 'FlinkSql', 'add-customjar',
...
+values (4, -1, '认证中心', '/auth', null, 'auth', 'SafetyCertificateOutlined', 'M', 0, 79, '2023-08-11 14:06:52',
+        '2023-09-26 15:08:42', null);
+insert into dinky_sys_menu
+values (5, -1, '数据开发', '/datastudio', './DataStudio', 'datastudio', 'CodeOutlined', 'C', 0, 4,
+        '2023-08-11 14:06:52', '2023-09-26 14:49:12', null);
+insert into dinky_sys_menu
+values (6, -1, '配置中心', '/settings', null, 'settings', 'SettingOutlined', 'M', 0, 115, '2023-08-11 14:06:53',
+        '2023-09-26 15:16:03', null);
+insert into dinky_sys_menu
+values (7, -1, '关于', '/about', './Other/About', 'about', 'SmileOutlined', 'C', 0, 143, '2023-08-11 14:06:53',
```
Comment (Zzm0809) [inline script/sql/dinky-pg.sql]: 这条记录先注释掉

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**